packaging/calculix-precice_2.17-1_amd64/usr/share/doc/calculix-precice/copyright

```diff
@@ -0,0 +1,9 @@
+Format: http://www.debian.org/doc/packaging-manuals/copyright-format/1.0/
+Upstream-Name: preCICE
+Upstream-Contact: preCICE mailing-list <info@precice.org>
+Copyright: (c) 2019 the preCICE developers <info@precice.org>
+Source: https://github.com/precice/precice
+
+Files: *
+Copyright: (c) 2019 the preCICE developers <info@precice.org>
+License: LGPL-3
```

Comment:
Why **L**GPL? This is the license of preCICE, the adapter currently is the regular GPL.